docs/source/examples/Layout and Styling.ipynb

```diff
@@ -0,0 +1,1892 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Layout and Styling of Jupyter widgets\n",
+    "\n",
+    "This notebook presents how to layout and style Jupyter interactive widgets to build rich and *reactive* widget-based applications."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## The `layout` attribute.\n",
+    "\n",
+    "Every Jupyter interactive widget has a `layout` attribute exposing a number of css properties that impact how widgets are laid out.\n",
+    "\n",
+    "### Exposed css properties\n",
+    "\n",
+    "<div class=\"alert alert-info\" style=\"margin: 20px\">\n",
+    "The following properties carry to the values of the css properties of the same name (underscores being replaced with dashes), applied to the top DOM elements of the corresponding widget.\n",
+    "</div>\n",
+    "\n",
+    "\n",
+    "** Sizes **\n",
+    "- `width`\n",
+    "- `height`\n",
```

Comment:
nit picking, sorry, but maybe change the ordering to height then width like the max/min props that follow